packages/fuels-core/src/json_abi.rs

```diff
@@ -421,6 +437,108 @@ impl ABIParser {
         Ok(Token::Array(result))
     }
 
+    /// Creates a tuple `Token` from an array of parameter types and a string of values.
```

Comment:
Is it a `Token::Tuple` or a "tuple `Token`"

Reply:
i put in Token::Tuple now to be exact. And I like consistency so I changed the other docstrings as well. There was an wrong docstring for the `tokenize_array` which was merged before. I changed it to match the function